### PR TITLE
Add RStudio daily build cask

### PR DESCRIPTION
**Pull request by noamross**
The RStudio daily build is the fresher, more dangerous version of rstudio-preview

### CODE REVIEW

File: Casks/rstudio-daily.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+cask :v1 => 'rstudio-daily' do
+  version :latest
+  sha256 :no_check
+
+  # rstudio.org is the official download host per the vendor homepage
```
Comment (vitorgalvao) [inline Casks/rstudio-daily.rb]: This isn’t required when the host of `homepage` and `url` are the same. Please remove it.

**Comment by vitorgalvao** [comment]
@fanquake **cask request** applies to issues only. To requests, not actual contributions. This is a **cask** **enhancement**.

**Comment by noamross** [comment]
I removed the comment per request.

**Comment by Ngrd** [comment]
Thank you. Could you please [squash your commits](http://davidwalsh.name/squash-commits-git)?

**Comment by noamross** [comment]
Done!

**Comment by Ngrd** [comment]
Thank you for the contribution.

**Pull request merged**